packages/gafl-webapp-service/src/handlers/__tests__/attribution-handler.spec.js

```diff
@@ -75,6 +87,54 @@ describe('The attribution handler', () => {
     delete process.env.ATTRIBUTION_REDIRECT
   })
 
+  it('redirects to RENEWAL_BASE if journey is renewal', async () => {
+    const query = {
+      [UTM.CAMPAIGN]: 'renewals',
+      [UTM.MEDIUM]: 'click_bait',
+      [UTM.CONTENT]: 'eieioh',
+      [UTM.SOURCE]: 'tomato',
+      [UTM.TERM]: 'Michaelmas'
+    }
+    const responseToolkit = generateResponseToolkitMock()
+
+    await attributionHandler(generateRequestMock(query), responseToolkit)
+    expect(responseToolkit.redirect).toHaveBeenCalledWith(RENEWAL_BASE)
+  })
+
+  it('redirects begins with { RENEWAL_BASE } when journey is renewal', async () => {
+    const query = {
+      [UTM.CAMPAIGN]: 'renewals',
+      [UTM.MEDIUM]: 'click_bait',
+      [UTM.CONTENT]: 'eieioh',
+      [UTM.SOURCE]: 'tomato',
+      [UTM.TERM]: 'Michaelmas'
+    }
+    const responseToolkit = generateResponseToolkitMock()
+    const regExMatch = new RegExp(`${RENEWAL_BASE}`)
+
+    await attributionHandler(generateRequestMock(query), responseToolkit)
+    expect(responseToolkit.redirect).toHaveBeenCalledWith(expect.stringMatching(regExMatch))
+  })
+
+  it.each([
+    ['B2F11U'],
+    ['AH56F6'],
+    ['GH330P']
+  ])('test renewal includes reference number when journey is renewal and reference number exists', async licenceKey => {
+    const query = {
+      [UTM.CAMPAIGN]: 'renewals',
+      [UTM.MEDIUM]: 'click_bait',
+      [UTM.CONTENT]: 'eieioh',
+      [UTM.SOURCE]: 'tomato',
+      [UTM.TERM]: 'Michaelmas',
+      [QUERYSTRING_LICENCE_KEY]: licenceKey
+    }
+    const responseToolkit = generateResponseToolkitMock()
+    await attributionHandler(generateRequestMock(query), responseToolkit)
+    const regExMatch = new RegExp(`^\\${RENEWAL_BASE}\\/${licenceKey}$`)
```

Comment:
This looks wrong, I think it should be 
```suggestion
    const regExMatch = new RegExp(`^${RENEWAL_BASE}/${licenceKey}$`)
```